docs/meshstack.billing.md

```diff
@@ -239,7 +239,31 @@ The following configuration options are available at `mesh.kraken.productcatalog
 
 Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
 
-The attributes that shall be part of the billing info on the chargeback statements can be configured as follows.
+The attributes that shall be part of the billing info on the chargeback statements can be configured.
+
+### Available metadata keys
+
+The following metadata keys are derived from meshStack metadata and therefore available on every meshStack implementation
+
+| Key                   | Description                                      |
+|-----------------------|:-------------------------------------------------|
+| tenantLandingZone     | Name of the Landing Zone used by the Tenant      |
+| last_modified         | Date when metadata was last modified             |
+| contactMail           | email address of the project owner               |
+| ownerUsername         | Username of the project owner                    |
```

Comment:
I think the "owner" is confusing as we will introduce the customer owner.

Reply:
The `ownerUsername` tag is available on every meshStack. When a customer owner is introduced, this should be changed accordingly in this document.